fabtests/regression/sighandler_test.c

```diff
@@ -78,6 +78,8 @@ int main(int argc, char **argv)
 				return EXIT_FAILURE;
 			}
 		}
+		hints->caps = FI_MSG;
+		hints->mode = FI_CONTEXT;
```

Comment:
It's better to set these prior to parsing the command line options, in case the options want to change them.

Reply:
I went back and forth but did this because all the other tests set them after for caps/modes that are required for the test so the user can't change them and make it fail.